README.md

```diff
@@ -95,25 +95,26 @@ npm t
 ```
 
 #### End-to-end tests
-In order for tests under the make target `test-integration-coordinator` to work, you must have built and be running the coordindator locally. In a seperate shell run:
+New examples can be added in the relevant directory under `models/primary-care|secondary-care`
 
-```
-make build
-make run-coordinator
-```
+Following the convention:
 
-Once the coordinator is up and displaying the port number, in another shell run:
+`{number}-{endpoint}-{request|response}-{?operation}-{status}.{ext}`
+
+Number is a way to group requests and responses together in each directory, for example in the below 
```

Comment:
Currently, all the examples have a number of 1, is this correct? For example, in `homecare/acute/no-nominated-pharmacy` there are 6 messages with a number of 1

Reply:
yep, it's a way to group examples together, i.e. a convert response for a send request, or prepare and send requests which are the same except for provenance, just we only have one example group for each at the moment

Reply:
ahh ok, think I understand. So the number isn't used in the smoke tests at all? I'll have to wait till I get there and see what it looks like :+1: